rootfs/etc/profile.d/banner.sh

```
@@ -43,14 +45,17 @@
 		fi
 		if [ -n "${BANNER}" ]; then
 			if [ "$BANNER_COMMAND" == "figlet" ]; then
+				local color_off="$(tput op 2>/dev/null)" # reset foreground and background colors to defaults
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)